.github/workflows/release-airgap-artifacts.yaml

```diff
@@ -31,12 +31,33 @@ jobs:
           aws-region: us-west-2
 
       - name: make release.chart-bundle
+        env:
+          DKP_OR_NKP_PREFIX: nkp
+        run: make ci.docker.run GOOS=linux GOARCH=amd64 RUN_WHAT="make release.chart-bundle"
+
+      - name: make release.chart-bundle (DKP)
```

Comment:
why do we need to keep dkp prefixed artifacts in 2.12+ releases?

Reply:
our team decided to ship both artifacts at least for this release, to avoid breaking any downstream tools/clients (or customers still using dkp artifacts) that we are not aware of using dkp artifacts. It is more of an interim thing to release both artifacts for this release, then we can remove dkp artifacts in the following release with more time to catch any downstream effects.

Reply:
The team feels more comfortable with releasing at the last minute. The reason is to keep CI from breaking at the last minute before a release.